### PR TITLE
Use stable wrappers in f32/f64::signum

### LINKED ISSUE #61638: Unnecessary usage of `intrinsics::copysignf32/64` in `signum`
In `f32::signum` and `f64::signum` we use `intrinsics::copysignf32` and `intrinsics::copysignf64`. However, these intrinsics are exposed with a stable wrapper `f32::copysign` and `f64::copysign`. We should use those instead.

https://github.com/rust-lang/rust/blob/d132f544f9d74e3cc047ef211e57eae60b78e5c5/src/libstd/f32.rs#L191

https://github.com/rust-lang/rust/blob/d132f544f9d74e3cc047ef211e57eae60b78e5c5/src/libstd/f64.rs#L169

### PR DESCRIPTION
**Pull request by JohnTitor**
Fixes #61638

r? @Centril

### CODE REVIEW

File: src/libstd/f32.rs [hunk 1]
```diff
@@ -188,7 +188,7 @@ impl f32 {
         if self.is_nan() {
             NAN
         } else {
-            unsafe { intrinsics::copysignf32(1.0, self) }
```
Comment (Centril) [inline src/libstd/f32.rs]: Note that `copysign` is a method so `1.0_f32.copy_sign(self)` would work.

Comment (JohnTitor) [reply src/libstd/f32.rs]: I fixed code following your comment.

**Comment by Centril** [comment]
Thanks! @bors r+ rollup

**Comment by RalfJung** [comment]
@rust-lang/infra why does bors call this PR "pending" at https://buildbot2.rust-lang.org/homu/queue/rust?

**Comment by pietroalbini** [comment]
Because bors was redeployed and its internal state got messed up -- again. I'll hit synchronize once the current PR finishes testing.

**Pull request merged**